### ISSUE TITLE
Need to hit close button in title bar twice to exit

### ISSUE THREAD

**Issue by trent-** [adobe/brackets#10774]
OS: Ubuntu 14.04 x86_64; Brackets version 1.2.0-15697

It seems to exit out of brackets, when using the close button on the title bar, I need to click it twice.

![demo](https://cloud.githubusercontent.com/assets/1747643/6763333/7b3adf98-cfcb-11e4-8abc-1cb3742c147b.gif)

* **Scenario 1:** Open brackets, all changes are saved, need to click the close button twice.
* **Scenario 2:** Open brackets, make some changes (without saving), click close - prompted to save changes. Click save - window remains open. Click close again, and brackets closes.
* **Scenario 3:** Open brackets, make no changes, click close. Window remains open. Make some changes (without saving) and click the close button again. Brackets closes without prompting me to save changes

**Comment by prafulVaishnav** [comment]
Duplicate of #4611

**Comment by prksingh** [comment]
As mentioned by @prafulVaishnav this may be a duplicate of #4611 Please note that there may be a fix for this in PR: https://github.com/adobe/brackets-shell/pull/458 Can you test if pulling those changes fixes your issue. There may be some testing pending on the Pull request. It would help us if you can confirm it fixes your issue. Thanks!

**Comment by prksingh** [comment]
Closing the issue to avoid duplication
